modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -9966,7 +9966,7 @@ Accessible getAccessible() {
         if (accessible == null) {
             accessible = Application.GetApplication().createAccessible();
             accessible.setEventHandler(new Accessible.EventHandler() {
-                @SuppressWarnings({"removal","deprecation"})
+                @SuppressWarnings("removal")
```

Comment:
Was this "deprecation" already useless before this change?

Reply:
Yes. The code to which it applied was removed back in JDK 9, but we forgot to remove the annotation.